src/components/inputs/reactHookForm/text/TextInput.tsx

```diff
@@ -110,7 +113,14 @@ export function TextInput({
                 adornment && {
                     handleClearValue,
                 })}
-            {...genHelperPreviousValue(previousValue!, adornment)}
+            helperText={
+                <HelperPreviousValue
+                    previousValue={previousValue}
+                    isNodeBuilt={isNodeBuilt}
+                    disabledTooltip={disabledTooltip || (!isUpdate && isNodeBuilt)}
+                    adornmentText={adornment?.text}
+                />
+            }
```

Comment:
```suggestion
            helperText={ previousValue !== undefined ?
                <HelperPreviousValue
                    previousValue={previousValue}
                    isNodeBuilt={isNodeBuilt}
                    disabledTooltip={disabledTooltip || (!isUpdate && isNodeBuilt)}
                    adornmentText={adornment?.text}
                /> : null
            }
```
as suggestion of Mathieu is not bad

Reply:
i ll take @Mathieu-Deharbe  suggestion. I think there is no need to add a check here since it is done in the component